source/includes/_v2_endpoints.md

```diff
@@ -1441,3 +1441,79 @@ the query.
 ### Returns
 
 An array of results.  Consult the [Result Type Dictionary](/api/#result-type-dictionary) for field descriptions.
+
+
+
+## Post Results to Custom Sources
+
+```ruby
+require 'rest_client'
+
+# Post a link
+
+RestClient.post 'https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post', {:feed_name => "Internal Feed", :type => 'link', :url => 'http://shoryuken.com/2015/09/03/japans-ruu-organizes-guilty-gear-round-robin-broadcast-for-american-players/', :email => 'admin@backstit.ch'}
+
+# Post an image
+
+RestClient.post 'https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post', {:feed_name => "Internal Feed", :type => 'image', :image => 'http://user-topic-images.s3.amazonaws.com/banner/example_image.jpg', :description => 'Example Image', :email => 'admin@backstit.ch'}
```

Comment:
use a link to our logo here so that we don't show our file structure and this is a valid link.